content/patterns/telco-hub/getting-started.adoc

```diff
@@ -0,0 +1,100 @@
+---
+title: Getting Started
+weight: 10
+aliases: /telco-hub/getting-started/
+---
+
+:toc:
+:imagesdir: /images
+:_mod-docs-content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
+[id="deploying-the-telco-hub-pattern"]
+= Deploying the Telco hub pattern
+
+
+[id="telco-hub-prerequisites"]
+== Prerequisites
+
+* An {ocp} 4.14 or later cluster with:
+
+** cluster administrator privileges.
+** compact cluster with at least 3 worker nodes.
+** more than 500 GB of available storage.
+** internet access.
+
+* A Git repository to host your configuration files.
+* Access to a terminal with `oc` and `git` command-line tools installed.
+
+[id="preparing-for-deployment"]
+== Preparing for deployment
+
+.Procedure
+
+. Clone telco-hub-pattern repository from GitHub and navigate to the directory:
++
+[source,terminal]
+----
+$ git clone https://github.com/validatedpatterns-sandbox/telco-hub-pattern.git
+$ cd telco-hub-pattern
+----
+
+. Configure components by editing the file `kustomize/overlays/telco-hub/kustomization.yaml` to enable required components:
++
+[source,yaml]
+----
+# =============================================================================
+# Telco Hub pattern - Kustomization Configuration
+# =============================================================================
```

Comment:
The pattern now has the required elements included by default. It also includes the optional Local Storage Operator and ODF. These are optional if the user has their own external storage (preferred) but are included in the pattern to provide an "all in one" solution. This part needs editing only if additional optional components are needed.